Content.Shared/GameObjects/Verbs/GlobalVerb.cs

```diff
@@ -27,6 +27,13 @@ public abstract class GlobalVerb
         /// <returns>The text string that is shown in the right click menu for this verb.</returns>
         public abstract string GetText(IEntity user, IEntity target);
 
+        /// <summary>
+        ///     Gets the category of this verb.
+        /// </summary>
+        /// <param name="user">The entity of the user opening this menu.</param>
+        /// <returns>The category of this verb.</returns>
+        public abstract string GetCategory(IEntity user, IEntity target);
```

Comment:
This should really just be a virtual so it can default to `""`. All the changes to existing verbs are annoying.

Reply:
Done. I thought about it at first, but in the end I opted for making it abstract for consistency with the rest of the class. Having it virtual is much more convenient though so I changed it